tests/installation/handle_reboot.pm

```diff
@@ -20,7 +20,7 @@ use utils qw(reconnect_mgmt_console);
 
 sub run {
     if (is_remote_backend) {
-        record_info 'Reconnect mgmt console';
+        record_info 'Remote', 'Reconnect mgmt console';
```

Comment:
Is there some warning in original version? If not and if whole string `Reconnect mgmt console` is visible without clicking in original version then I'd use the original version.

Reply:
the signature of record_info takes $title and $description. Without you see `Use of uninitialized value $_[0] in join or string at...` i am not sure if title is the right one but the description is also visible hovering over the frame.